phpstan-baseline.neon

```diff
@@ -1,12 +1,22 @@
 parameters:
 	ignoreErrors:
+		-
+			message: "#^Call to an undefined method Symfony\\\\Component\\\\HttpFoundation\\\\RequestStack\\:\\:getMasterRequest\\(\\)\\.$#"
+			count: 1
+			path: src/Knp/Menu/Matcher/Voter/RouteVoter.php
+
+		-
+			message: "#^Strict comparison using \\=\\=\\= between false and array\\<Knp\\\\Menu\\\\ItemInterface\\> will always evaluate to false\\.$#"
```

Comment:
this one is because `array_combine` returns `false` only on PHP 7, not on PHP 8 (on PHP 8, it throws an exception on failure)